net/http/authz/store.go

```diff
@@ -9,28 +9,29 @@ import (
 	"chain/errors"
 )
 
+const keyPrefix = `/core/grant/`
```

Comment:
Minor, but I feel like this actual path is specific to chain core, whereas this package could conceivably be used in other products. (Consider if signerd had a database, maybe it would put grants in `/signerd/grant/`.)

This isn't as bad as actually importing a package in `chain/core/...`; it's just a constant string and not hard to change later if necessary. So okay. But maybe we should have a todo about factoring this out?

Ideally imo we would instantiate the grant Store in just one place, and that place would dictate the path prefix. Currently it seems like that would require an annoying amount of plumbing between chain/cmd/cored and chain/core and chain/core/config, so I'm fine with not doing that for now.